pkg/etcd/etcd.go

```diff
@@ -65,7 +65,7 @@ const (
 	endpoint            = "https://127.0.0.1:2379"
 	testTimeout         = time.Second * 10
 	manageTickerTime    = time.Second * 15
-	learnerMaxStallTime = time.Minute * 1
+	learnerMaxStallTime = time.Minute * 5
```

Comment:
This seems relatively high, is it the recommended value from the etcd folks? Or is this debugging cruft?

Reply:
no it was a problem when etcd delays to start, it happens for multiple reasons, but in my testing it reached about 3 minutes until etcd started on a second node, I talked with @brandond about it and we agreed to increase the stall time, but we didnt agreed on a specific number